packit_service/constants.py

```diff
@@ -14,9 +14,7 @@
 
 CONFIG_FILE_NAME = "packit-service.yaml"
 
-TESTING_FARM_TRIGGER_URL = (
-    "https://scheduler-testing-farm.apps.ci.centos.org/v0/trigger"
-)
+TESTING_FARM_API_URL = "https://api.dev.testing-farm.io/v0.1/"
```

Comment:
"dev" - doesn't sound like a prod deployment, I wonder if there will be two or more

Reply:
Yes, @thrix said that it will run here for about half a year and then move to `api.testing-farm.io` when it's time. He'll let us know when prod API is ready.

Reply:
thanks for info